_backend/log-echo.php

```diff
@@ -3,15 +3,19 @@
 require_once __DIR__ . '/bootstrap.php';
 
 // Setup sentry error logging
-if (isset($config['sentry_key']) && $config['sentry_key'] !== false) {
+if (
+    getenv('PHPENV') == 'production' &&
```

Comment:
I'm like 99% sure that all the `beta.elementary.io` sites are also in production to avoid weird errors while testing.